purl-dev/schema.xml

```diff
@@ -14,6 +14,14 @@
 
     <field name="id" type="string" stored="true" indexed="true" multiValued="false" required="true"/>
     <field name="timestamp" type="date" indexed="true" stored="true" default="NOW" multiValued="false"/>
+    <copyField source="*_s" dest="*_ssim"/>
+    <copyField source="*_t" dest="*_tsim"/>
+  
+    <!-- Temporary Dynamic Fields that mimic the Argo schema to allow Purl-Fetcher fields to Match Dor-Fetcher fields -->
+     <!-- Following the upgrade of Argo-prod these will be delated and _ssim/_tsim will be used -->
+    <dynamicField name="*_s"  type="string"  indexed="true"  stored="true" multiValued="true"/>
```

Comment:
@cbeer 

For the moment I need the _t and _s fields to be indexed for searching.  There is one shared piece of code that searches both the argo solr core and this core, which StanfordSync will use.  So to search Argo it needs to form a query including say the field 'is_member_of_collection_s' since the Argo core has no _ssim or _tsim.

Once Joe has completed his upgrade of Argo Solr and rebuilt the Argo core that will become a _ssim field.  So we want to avoid going back and updating the code to have the logic to determine whether it should be using is_member_of_collection_s or is_member_of_collection_ssim, so for now both Argo and StanfordSync will use the _s option.  When the Argo core is rebuilt I'll update this schema to remove _s, _t and the copyfields as both with now use _ssim.  For now though the copyfields are just there to duplicate the data and avoid needing to touch the solr documents to move from _s and _t to _ssim and _tsim when that day comes and the code will start running its queries using _ssim and _tsim endings as opposed to _s and _t.

 The two fields in question I'll be duplicating are "is_member_of_collection_s" and "identityMetadata_objectType_t".  If you want to avoid the growth in size brought on by duplication, I can drop the copyfields and just move the data later.